observers/java/src/org/openda/observers/SerializableObservationDescriptions.java

```diff
@@ -49,7 +49,7 @@
 	int propertyCount;
 	int observationCount;
 	ITime[] times;
-	List<IPrevExchangeItem> exchangeItems;
+	List<IExchangeItem> exchangeItems;
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=5102854)